geojson-rewind.js

```diff
@@ -1,11 +1,13 @@
 #!/usr/bin/env node
 
-var rewind = require('./'),
-    getStream = require('get-stream'),
-    fs = require('fs'),
-    argv = require('minimist')(process.argv.slice(2), {
-        boolean: 'clockwise'
-    });
+import rewind from "@mapbox/geojson-rewind";
```

Comment:
Why not import local module?

Reply:
It is the local module :)

This is a quirk/benefit of how [the `exports` field](https://nodejs.org/docs/latest-v12.x/api/packages.html#packages_package_entry_points) in the `package.json` file works. It's a little safer because it allows the entry point to be renamed without breaking. I've done the same in the test module. Unfortunately `import rewind from "./"` doesn't seem to work, so the direct translation from CJS isn't an option (at least, I haven't managed to reproduce that behaviour).

The alternative is the direct local import, which would look like `import rewind from "../index.js";`. Happy to change to that if you prefer, though it is (very slightly) less robust.